src/Components/CreateImageWizard/formComponents/ReviewStep.js

```diff
@@ -315,22 +317,34 @@ const ReviewStep = () => {
                             <br />
                             If using an activation key with command line
                             registration, you must provide your
-                            organization&apos;s ID.
+                            organization&apos;s ID.{' '}
+                            {getState()?.values?.[
+                              'subscription-organization-id'
+                            ] !== undefined ? (
+                              `Your organization's ID is ${
+                                getState()?.values?.[
+                                  'subscription-organization-id'
+                                ]
+                              }`
+                            ) : (
+                              <Spinner />
```

Comment:
If you disable the line of code that sets `'subscription-organization-id'` so you can see what happens if it is `undefined`, the popover looks like this:

![spinner](https://user-images.githubusercontent.com/95542540/204760885-2c17751a-59e3-4c0d-8fc5-b82a05393c2c.gif)

What if we tried something like `Your organization's ID is <Spinner size="sm" />.`?